examples/examples

```diff
@@ -0,0 +1 @@
+examples
```

Comment:
This is a spurious file that should not be part of the commit.  It gets (incorrectly) created by the install program.  We should eventually fix that, but for now you should remove it from your commit (`git rm examples/examples`).